source/aggregation/agg-exp-ops.txt

```diff
@@ -0,0 +1,1183 @@
+     - :manual:`$maxN </reference/operator/aggregation/maxN-array-element/>`
+
+   * - `min() <{+core-api+}/com/mongodb/client/model/mql/MqlArray.html#min(T)>`__
+     - :manual:`$min </reference/operator/aggregation/min/>`
+
+   * - `minN() <{+core-api+}/com/mongodb/client/model/mql/MqlArray.html#minN(com.mongodb.client.model.mql.MqlInteger)>`__
+     - :manual:`$minN </reference/operator/aggregation/minN-array-element/>`
+
+   * - `multiply() <{+core-api+}/com/mongodb/client/model/mql/MqlArray.html#multiply(java.util.function.Function)>`__
+     - :manual:`$multiply </reference/operator/aggregation/multiply/>`
+
+   * - `size() <{+core-api+}/com/mongodb/client/model/mql/MqlArray.html#size()>`__
+     - :manual:`$size </reference/operator/aggregation/size/>`
+
+   * - `slice() <{+core-api+}/com/mongodb/client/model/mql/MqlArray.html#slice(int,int)>`__
+     - :manual:`$slice </reference/operator/aggregation/slice/>`
+
+   * - `sum() <{+core-api+}/com/mongodb/client/model/mql/MqlArray.html#sum(java.util.function.Function)>`__
+     - :manual:`$sum </reference/operator/aggregation/sum/>`
+
+   * - `union() <{+core-api+}/com/mongodb/client/model/mql/MqlArray.html#union(com.mongodb.client.model.mql.MqlArray)>`__
+     - :manual:`$setUnion </reference/operator/aggregation/setUnion/>`
+
+   * - `unionArrays() <{+core-api+}/com/mongodb/client/model/mql/MqlArray.html#unionArrays(java.util.function.Function)>`__
+     - :manual:`$setUnion </reference/operator/aggregation/setUnion/>`
+
+Suppose you have a collection of movies, each of which contains an array
+of nested documents for upcoming showtimes. Each nested document
+contains an array that represents the total number of seats in the
+theater, where the first array entry is the number of premium seats and
+the second entry is the number of regular seats. Each nested document
+also contains the number of tickets that have already been bought for
+the showtime. A document in this collection might resemble the
+following:
+
+.. code-block:: json
+   :copyable: false
+
+   {
+     "_id": ...,
+     "movie": "Hamlet",
+     "showtimes": [
+       {
+         "date": "May 14, 2023, 12:00 PM",
+         "seats": [ 20, 80 ],
+         "ticketsBought": 100
+       },
+       {
+         "date": "May 20, 2023, 08:00 PM",
+         "seats": [ 10, 40 ],
+         "ticketsBought": 34
+       }]
+   }
+
+The ``filter()`` method displays only the results matching the provided
+predicate. In this case, the predicate uses ``sum()`` to calculate the
+total number of seats and compares that value to the number of ``ticketsBought``
+with ``lt()``. The ``project()`` method stores these filtered results as a new
+``availableShowtimes`` array.
+
+.. tip::
+
+   You must specify the type of the array that you retrieve with the
+   ``getArray()`` method if you work with the values of the
+   array as their specific type.
```

Comment:
Q: Little unclear on the meaning of the "if you work with the values of the array as their specific type". Is there any way to reword this?